esp/applications/eclservices/esp.yaml

```diff
@@ -14,14 +14,13 @@ esp:
    logDir: "-"
 
    tls_config:
-     certificate: /etc/HPCCSystems/certificates/{$instance}/server.crt
-     privatekey: /etc/HPCCSystems/certificates/{$instance}/private.key
-     cipherList:
-       verify:
-          enable: false
-          address_match: false
-          accept_selfsigned: true
-          ca_certificates:
-            - path: "ca.pem"
-          trusted_peers:
-            - anyone
+     certificate: /opt/HPCCSystems/secrets/certificates/internal/tls.crt
+     privatekey: /opt/HPCCSystems/secrets/certificates/internal/tls.key
+     cipherList: "ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5"
+     verify:
+       enable: true
```

Comment:
This flag basically turns on mutual TLS.  Since this is an internal service, if TLS is on it should be mutual TLS.  On the other hand we don't want to require client certificates for public services.

Reply:
I'll add a comment since it can be confusing.